src/server/api/routers/metric.ts

```
@@ -649,11 +649,25 @@ export const metricRouter = createTRPCRouter({
             );
             break;
           case "posthog":
+            // For Query API POST requests, provide example body based on endpoint label
+            let postBody = undefined;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)